src/plugins/intel_gpu/src/graph/impls/ocl/ctc_loss.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ctc_loss/ctc_loss_kernel_ref.hpp"
+#include "ctc_loss/ctc_loss_kernel_selector.hpp"
+#include "ctc_loss_inst.hpp"
+#include "impls/implementation_map.hpp"
+#include "primitive_base.hpp"
+
+namespace cldnn {
+namespace ocl {
+
+struct ctc_loss_impl : typed_primitive_impl_ocl<ctc_loss> {
+    using parent = typed_primitive_impl_ocl<ctc_loss>;
+    using parent::parent;
+
+    std::unique_ptr<primitive_impl> clone() const override {
+        return make_unique<ctc_loss_impl>(*this);
+    }
+
+    static primitive_impl* create(const ctc_loss_node& arg, const kernel_impl_params& impl_param) {
+        auto params = get_default_params<kernel_selector::ctc_loss_params>(impl_param);
+        auto optional_params =
+            get_default_optional_params<kernel_selector::ctc_loss_optional_params>(arg.get_program());
+
+        auto primitive = arg.get_primitive();
```

Comment:
Please get rid of usage of arg (program_node) in the new impl creator. 
It will be deprecated in the future. You can use typed_desc of impl_param.

Reply:
Done.